### ISSUE TITLE
UI: Confirmation button disappeared from starting multiple instance by non-admin user

### ISSUE THREAD

**Issue by levindecaro** [apache/cloudstack#9651]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.19.1.1
~~~

##### CONFIGURATION

N/A

##### OS / ENVIRONMENT

CentOS 7
KVM

##### SUMMARY

After upgrade from 4.19.0 to 4.19.1.1, when a regular user select multiple vm to power on, the confirmation button is missing, however, no such issue when using global admin.

##### STEPS TO REPRODUCE

~~~
1. Login as a regular user
2. go to instance page
3. Tick one VM
4. Click [Play] button to start the instance
~~~

##### EXPECTED RESULTS

~~~
Display [OK] and [Cancel] button in the start instance dialog.
~~~

##### ACTUAL RESULTS

~~~
[OK] and [Cancel] button are missing.
~~~

![image](https://github.com/user-attachments/assets/761d42ff-908d-4c58-8a5e-cc6119c4584f)

**Comment by bernardodemarco** [comment]
Fixed by #9652. @levindecaro, feel free to reopen this issue if necessary.

**Comment by levindecaro** [comment]
@bernardodemarco thank you for your prompt action, i'd like to confirm the bug is fixed.
